DPGAnalysis/SiStripTools/python/apvcyclephaseproducerfroml1ts_cfi.py

```diff
@@ -9,5 +9,9 @@
                                                                ),
                            defaultPhases = cms.vint32(30,30,30,30),
                            l1TSCollection = cms.InputTag("scalersRawToDigi"),
+                           tcdsRecordLabel= cms.InputTag("unpackTcds","tcdsRecord"),
```

Comment:
forgot to mention, but here too the source of TCDS digis needs to be modified. Sorry for not pointing it out earlier.

Reply:
No problem, sorry I should have remembered it was set also here